_people/jongkyu_kim.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jongkyu Kim
+category: Alumni
+position: some Alumni
```

Comment:
some Alumni? Are we changing that later? Sounds weird

Reply:
I would just remove the position